hc/api/tests/test_create_check.py

```diff
@@ -44,26 +46,27 @@ def test_it_works(self):
         self.assertEqual(check.tags, "bar,baz")
         self.assertEqual(check.timeout.total_seconds(), 3600)
         self.assertEqual(check.grace.total_seconds(), 60)
+        self.assertEqual(check.last_ping, None)
+        self.assertEqual(check.n_pings, 0)
 
     def test_it_accepts_api_key_in_header(self):
         payload = json.dumps({"name": "Foo"})
 
         ### Make the post request and get the response
-        r = {'status_code': 201} ### This is just a placeholder variable
-
-        self.assertEqual(r['status_code'], 201)
+        r = self.post({"api_key": "abc"})
+        self.assertEqual(r.status_code, 201)
 
     def test_it_handles_missing_request_body(self):
         ### Make the post request with a missing body and get the response
-        r = {'status_code': 400, 'error': "wrong api_key"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "wrong api_key")
+        r = self.post({}, expected_error="wrong api_key")
+        self.assertEqual(r.status_code, 400)
+        self.assertEqual(r.json()['error'], 'wrong api_key')
 
     def test_it_handles_invalid_json(self):
         ### Make the post request with invalid json data type
-        r = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "could not parse request body")
+        r = self.client.post(self.URL, {"api_key": "abc"}, content_type="application/json")
+        self.assertEqual(r.status_code, 400)
+        self.assertEqual(r.json()['error'], "could not parse request body")
 
     def test_it_rejects_wrong_api_key(self):
         self.post({"api_key": "wrong"},
```

Comment:
@Jak-Sidious @pndemo where are the tests for these:
### Test for the assignment of channels
### Test for the 'timeout is too small' and 'timeout is too large' errors

Reply:
@WNjihia Noted. Working on fixing this.